Sources/Basic/FSProxy.swift

```diff
@@ -268,19 +268,19 @@ private class LocalFS: FSProxy {
 //
 // FIXME: This class does not yet support concurrent mutation safely.
 public class PseudoFS: FSProxy {
-    private class Node {
+    fileprivate class Node {
```

Comment:
Ah, I remember now.  The rule for `private` is the inner scope can refer to `private` members in the outer scope but not vice-versa.